lookout/style/format/model.py

```diff
@@ -91,14 +93,16 @@ def disassemble_rule(rule: Rule) -> tuple:
             features = numpy.fromiter(features, numpy.uint16, rule_len)
             cmps = numpy.fromiter(cmps, numpy.bool, rule_len)
             thresholds = numpy.fromiter(thresholds, numpy.float32, rule_len)
-            return rule.stats.cls, rule.stats.conf, features, cmps, thresholds, rule_len
+            return (rule.stats.cls, rule.stats.conf, rule.stats.support, features, cmps,
+                    thresholds, rule_len)
 
         disassembled_rules = list(zip(*[disassemble_rule(rule) for rule in rules]))
         return dict(
             cls=numpy.array(disassembled_rules[0], dtype=numpy.uint16),
             conf=numpy.array(disassembled_rules[1], dtype=numpy.float32),
-            features=numpy.concatenate(disassembled_rules[2]),
-            cmps=numpy.concatenate(disassembled_rules[3]),
-            thresholds=numpy.concatenate(disassembled_rules[4]),
-            lengths=numpy.array(disassembled_rules[5], dtype=numpy.uint16),
+            support=numpy.array(disassembled_rules[2], dtype=numpy.uint16),
```

Comment:
Can you please append to the end here so that the code does not unnecessarily change. This also means changing the indices in the underlying format - ` disassembled_rules[6]` so that it does not look ugly and actually retains the backwards compatibility.

Reply:
It was merged in another PR https://github.com/src-d/style-analyzer/pull/193/files#diff-39d834adb41513661073cf812d4378ef already.